### PR TITLE
Merge remaining optargs into one branch

### PR DESCRIPTION
**Pull request by Jieran-S**
Add remaining optargs. Meanwhile:

1. Change wording for schema file
2. Add a contributor description for their implementation script.

Will create an issue for questions found upon adding optargs file.

### CODE REVIEW

**Review by shdam — CHANGES_REQUESTED** [review]
Delete bass_optargs.json to prevent merge conflicts with main.

File: method/bass/bass_optargs.json
[the remark this answers is not in the document]
Comment (Jieran-S) [reply method/bass/bass_optargs.json:1]: Aha the bass config is your commit I just copied it. But i can delete it

[the remark this answers is not in the document]
Comment (shdam) [reply method/bass/bass_optargs.json:1]: Ah, I'm not sure how that is handled when merging with main. Will it know it is the same changes?

[the remark this answers is not in the document]
Comment (Jieran-S) [reply method/bass/bass_optargs.json:1]: It should be fine but I deleted it just to be sure! :))

[the remark this answers is not in the document]
Comment (shdam) [reply method/bass/bass_optargs.json:1]: Hmm, I am curious to see how git handles creating and deleting a file that exists in main. Hopefully, all goes well.

[the remark this answers is not in the document]
Comment (Jieran-S) [reply method/bass/bass_optargs.json:1]: That's a good point...we can check on PR and if it got deleted we can always restore it

**Review by shdam — APPROVED** [review]

**Pull request merged**